packages/webapp/pages/onboarding.tsx

```diff
@@ -122,18 +143,11 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.EditTag);
     }
 
-    if (
```

Comment:
This was before it was rolled out to 100%?
(Can't recall)

Reply:
Ah ok no worries, thanks for the cleanup in that case :) 
Makes things a lot easier to follow.